eitprocessing/datahandling/sparsedata.py

```diff
@@ -13,38 +13,40 @@
 
 @dataclass(eq=False)
 class SparseData(Equivalence, SelectByTime):
-    """Container for data occuring at unpredictable time points.
+    """Container for data related to individual time points.
 
-    In sparse data the time points are not necessarily evenly spaced. Data can consist time-value pairs or only time
-    points. Values generally are numeric values in arrays, but can also be lists of different types of object.
+    Sparse data is data for which the time points are not necessarily evenly spaced. Data can consist time-value pairs
+    or only time points. Values generally are numeric values in arrays, but can also be lists of different types of
+    object.
```

Comment:
I don't think the comment on values adds clarity here, given that I don't think there are any true restrictions (neither programmatically nor in practice).

```suggestion
    or only time points.
```

Reply:
I think I added this here to contrast ContinuousData where `values` is expected to be a numpy array. But it is fine not to mention it specifically.